Package.resolved

```diff
@@ -1,6 +1,33 @@
 {
   "object": {
     "pins": [
+      {
+        "package": "Commander",
```

Comment:
where are we getting these from?

Reply:
I think because I referenced an old swift-grpc here.   Trying to fix up the merge conflicts, but git is fighting me.  Sigh.